source/platform/linux/he_bus/inc/he_bus_core.h

```diff
@@ -88,14 +88,17 @@ typedef he_bus_error_t (*he_bus_table_add_row_handler_t)(char const *tableName,
     char const *aliasName, uint32_t *instNum);
 typedef he_bus_error_t (*he_bus_table_remove_row_handler_t)(char const *rowName);
 typedef he_bus_error_t (*he_bus_method_handler_t)(char const *methodName,
-    he_bus_raw_data_t *inParams, he_bus_raw_data_t *outParams, void *asyncHandle);
+    he_bus_data_object_t *inParams, he_bus_data_object_t *outParams, void *asyncHandle);
 typedef he_bus_error_t (*he_bus_event_sub_handler_t)(char *eventName,
     he_bus_event_sub_action_t action, int32_t interval, bool *autoPublish);
 
 typedef he_bus_error_t (
     *he_bus_event_consumer_sub_handler_t)(char *event_name, he_bus_raw_data_t *p_data, void *userData);
 typedef he_bus_error_t (*he_bus_event_sub_ex_async_handler_t)(char *event_name, he_bus_error_t ret, void *userData);
 
+typedef void (*he_bus_method_async_resp_handler_t) (char const* methodName, he_bus_error_t ret,
+    he_bus_data_object_t *params, void *user_data);
```

Comment:
```suggestion
    he_bus_data_object_t *params, void *userData);
```
stick to the convention - in this file dominant is camel casing.